### PR TITLE
Provide default configuration for mutt.header_cache

### PR DESCRIPTION
**Pull request by nottwo**
In the spirit of providing sensible defaults, let's set mutt.header_cache.  Mutt 1.5.7 (2005) or later is required.

Some people might want to specify a directory for this (or maybe mutterer could autocreate this as a directory), but mutt will happily build the cache in a single file.

Since we're assuming an offlineimap setup, I didn't set message_cachedir for caching message bodies.

### CODE REVIEW

**Comment by mwnx** [comment]
Hmm, I'm not sure if I want to apply this one.

First of all, I must admit that I didn't even know about this feature of
mutt. Seems pretty useful actually, so thanks for introducing me.

However, it's not necessary for anything else to function, nor does mutterer
improve this feature in any way.

My policy so far −tell me if you find any counterexamples− has been to only
include in mutterer's presets entries that are either relevant to several
accounts at once, or relevant to several of the software components at once
(mutt, offlineimap, msmtp).

For any mutt configuration that could be done just as well before, I
encourage users to simply put the entries in ~/.mutt/muttrc. Alternatively,
adding an [accounts %defaults] section containing said entries in the
.muttererrc works too, but I find that to be messier myself.


In conclusion, unless you have some very convincing arguments ;), I'll be
closing this pull request.

**Comment by nottwo** [comment]
Fair enough.

I see mutterer having the potential for being a way of easing people into using
& configuring mutt.

Part of that is providing sensible defaults.  I also plan on opening a pull
request for configuring notmuch :)

How would you feel about these "non-essential" configurations getting their own
`presets/` files that users can source in their `.muttererrc` at their
convenience?

**Comment by mwnx** [comment]
Actually, I was going to propose that you could add a file in presets/ with
what you believe to be sensible defaults. For instance, I don't know,
"mutt-sensible-defaults"?

Also, I'm glad to hear you're working on notmuch integration. If you need
any help on that, don't hesitate to ask. There's also a graph
(doc/call-graph.{dot,png}) which might help you in understanding the
somewhat −and perhaps unnecessarily− convoluted structure of Mutterer.

By the way, feel free to promote Mutterer if you want; if you have a blog
for instance. I just posted an announcement on the mutt and offlineimap
mailing list, but that kind of work is not really my thing.
